src/components/side_panel/pivot/pivot_side_panel/pivot_spreadsheet_side_panel/pivot_spreadsheet_side_panel.xml

```diff
@@ -5,9 +5,15 @@
         <Section>
           <t t-set-slot="title">Name</t>
           <EditableName name="name" displayName="displayName" onChanged.bind="onNameChanged"/>
-          <div class="btn btn-link o_duplicate_pivot" t-on-click="duplicatePivot">
+          <div class="btn btn-link o_duplicate_pivot" t-on-click="() => this.duplicatePivot(false)">
             <i class="fa fa-solid fa-copy me-1"/>
-            Duplicate Pivot
+            Duplicate
+          </div>
+          <div
+            class="btn btn-link o_duplicate_pivot_in_new_sheet"
+            t-on-click="() => this.duplicatePivot(true)">
+            <i class="fa fa-solid fa-copy me-1"/>
+            Duplicate in a new sheet
```

Comment:
If I follow the spec on the task, the idea was to **replace** the behaviour of `Duplicate` by duplicating the pivot in a new sheet, not adding a second duplicate item. Or the spec has been discussed vocally and not updated ?

Reply:
@pro-odoo adapted the code 😉